.buildkite/pipeline.release-experimental.yml

```diff
@@ -0,0 +1,36 @@
+steps:
+  - wait
+
+  - name: ":s3: publish binaries"
+    command: "scripts/publish-to-s3.sh"
+    branches: "master 2-1-stable"
```

Comment:
The 2-1-stable stuff makes no sense in master, you can remove all those. Each stable release branch has it's own version: https://github.com/buildkite/agent/blob/2-6-stable/.buildkite/pipeline.yml#L100